### PR TITLE
[platform] Move ChangeMaxNumberOfOpenFiles to platform::tests_support, delete duplicates.

### PR DESCRIPTION
**Pull request by tatiana-yan**

### CODE REVIEW

File: routing/routing_integration_tests/routing_test_tools.cpp [hunk 1]
```diff
@@ -6,18 +6,19 @@
...
 #include "routing/router_delegate.hpp"
 #include "routing/routing_callbacks.hpp"
 
+#include "storage/country_parent_getter.hpp"
+#include "storage/routing_helpers.hpp"
+
 #include "indexer/data_source.hpp"
 
-#include "storage/country_parent_getter.hpp"
+#include "platform/platform_tests_support/change_number_of_open_files.hpp"
```
Comment (mpimenov) [inline routing/routing_integration_tests/routing_test_tools.cpp]: limits.hpp?

Comment (mpimenov) [reply routing/routing_integration_tests/routing_test_tools.cpp]: or utils/helpers?

**Comment by tatiana-yan** [comment]
JTALL

File: routing/routing_integration_tests/routing_test_tools.cpp (hunk 1 above)
Comment (tatiana-yan) [reply routing/routing_integration_tests/routing_test_tools.cpp]: ok, done, helpers

**Comment by tatiana-yan** [comment]
JTALL

**Comment by mapsmetest** [comment]
JTIOS

File: platform/platform_tests_support/helpers.hpp [hunk 2]
```diff
@@ -0,0 +1,17 @@
+#include <sys/resource.h>
```
Comment (mpimenov) [inline platform/platform_tests_support/helpers.hpp]: тоже под ifdef?

Comment (tatiana-yan) [reply platform/platform_tests_support/helpers.hpp]: done

**Pull request merged**